packages/testlib/jest.config.js

```diff
@@ -15,4 +15,5 @@ module.exports = {
 	modulePathIgnorePatterns: [
 		'<rootDir>/dist',
 	],
+	passWithNoTests: true
```

Comment:
Did you mean to leave this in?

Reply:
Yes since I deleted the only test file which was a noop, this is required.